core/src/main/java/org/elasticsearch/search/suggest/SuggestBuilder.java

```diff
@@ -119,6 +119,9 @@ public static SuggestBuilder fromXContent(QueryParseContext parseContext, Sugges
         String fieldName = null;
 
         XContentParser.Token token;
+        if (parser.currentToken() != XContentParser.Token.START_OBJECT) {
+            parser.nextToken();
+        }
```

Comment:
can we add an assert here `assert parser.currentToken() == XContentParser.Token.START_OBJECT`